testing/tests/book_example_performance_fmt_call_fast_writable.rs

```diff
@@ -0,0 +1,52 @@
+use std::fmt::{self, Write};
+
+use askama::NO_VALUES;
+use askama::filters::FastWritable;
+
+// In a real application, please have a look at
+// https://github.com/kdeldycke/awesome-falsehood/blob/690a070/readme.md#human-identity
+struct Name<'a> {
+    forename: &'a str,
+    surname: &'a str,
+}
+
+impl fmt::Display for Name<'_> {
+    // Because the method simply forwards the call, it should be `inline`.
```

Comment:
Normally, the compiler is supposed to do that on its own, especially for small functions. So might be worth saying that normally it's not really necessary because rustc does it on its own but that it doesn't hurt to have it.

Reply:
Not across crates units, or does it?

Reply:
Hum, not sure about that...